graphics/libmypaint/Portfile

```diff
@@ -23,8 +23,7 @@ long_description    libmypaint, a.k.a. \"brushlib\", is a library for making bru
 github.tarball_from releases
 use_xz              yes
 
-# Using the same python as MyPaint for consistency.
```

Comment:
please don't delete this comment, instead make sure to either update the Python version on both ports as the comment tells you ;)

Reply:
Ah, I thought the comment meant same as the upstream port, which explicitly maps out 3.8 and no further. However the build was fine so that's why I deleted the comment. I didn't realize it referred to the other port. Will make the comment more clear.